packages/framework/fluid-framework/src/legacy.ts

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// As a safety net against unintentionally exporting types tagged @internal,
+// we re-export from '/legacy' rathen than '/internal'.
```

Comment:
Nit typo:

```suggestion
// we re-export from '/legacy' rather than '/internal'.
```

Reply:
Feel free to ignore to save yourself the ci run if this is the only issue.

Reply:
My other PR needs to make another trip through CI.  I'll fix the type there.  Thanks!